doc/examples/animations_pulsating_sphere.py

```diff
@@ -0,0 +1,127 @@
+"""Animations of pulsating sphere."""
+import sfs
+import numpy as np
+from matplotlib import pyplot as plt
+from matplotlib import animation
+
+
+def update_frame_displacement(i, grid, displacement, phasor, scat, frames):
+    """Update frame of sound particle animation."""
+    position = (grid + displacement * phasor**i).apply(np.real)
+    position = np.column_stack([position[0].flatten(), position[1].flatten()])
+    scat = scat.set_offsets(position)
+    return scat
+
+
+def update_frame_velocity(i, velocity, phasor, quiv, frames):
+    """Update frame of particle velocity animation."""
+    quiv.set_UVC(*(velocity[:2] * phasor**i).apply(np.real))
+    return quiv
+
+
+def update_frame_pressure(i, grid, pressure, radius, amplitude, phasor,
+                          pulsate, im, frames):
+    """Update frame of sound pressure animation."""
+    distance = np.linalg.norm(grid)
+    p = pressure * phasor**i
+    if pulsate:
+        p[distance <= radius + amplitude * np.real(phasor**i)] = np.nan
+    im.set_array(np.real(p))
+    return im
+
+
+def animation_particle_displacement(omega, center, radius, amplitude, grid,
+                                    frames, figsize=(8, 8), **kwargs):
+    """Generate sound particle animation."""
+    velocity = sfs.mono.source.pulsating_sphere_velocity(
+               omega, center, radius, amplitude, grid)
+    displacement = sfs.util.displacement(velocity, omega)
+    phasor = np.exp(1j * 2 * np.pi / frames)
+
+    fig = plt.figure(figsize=figsize)
+    ax = plt.axes()
+    scat = sfs.plot.particles(grid + displacement, **kwargs)
+    ax.axis([grid[0].min(), grid[0].max(), grid[1].min(), grid[1].max()])
+    return animation.FuncAnimation(
+            fig, update_frame_displacement, frames,
+            fargs=(grid, displacement, phasor, scat, frames))
```

Comment:
You should probably pass `blit=True` and make it possible to pass in `interval` (but provide a meaningful default).

I guess `repeat` is not necessary, because we always want `repeat=True`, right?